src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/JudgeDraftOrderMidHandler.java

```diff
@@ -36,27 +36,27 @@ public JudgeDraftOrderMidHandler(FinremCaseDetailsMapper finremCaseDetailsMapper
     public boolean canHandle(CallbackType callbackType, CaseType caseType, EventType eventType) {
         return CallbackType.MID_EVENT.equals(callbackType)
             && CaseType.CONTESTED.equals(caseType)
-            && (EventType.JUDGE_DRAFT_ORDER.equals(eventType));
+            && EventType.JUDGE_DRAFT_ORDER.equals(eventType);
     }
 
     @Override
     public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(FinremCallbackRequest callbackRequest,
                                                                               String userAuthorisation) {
+        log.info(CallbackHandlerLogger.midEvent(callbackRequest));
         FinremCaseDetails caseDetails = callbackRequest.getCaseDetails();
-        String caseId = String.valueOf(caseDetails.getId());
-        log.info("Invoking contested event {} mid callback for Case ID: {}",
-            EventType.JUDGE_DRAFT_ORDER, caseId);
         FinremCaseData caseData = caseDetails.getData();
         DraftDirectionWrapper draftDirectionWrapper = caseData.getDraftDirectionWrapper();
-        List<DraftDirectionOrderCollection> draftDirectionOrderCollection = draftDirectionWrapper.getDraftDirectionOrderCollection();
+        List<DraftDirectionOrderCollection> judgeApprovedOrderCollection = draftDirectionWrapper.getJudgeApprovedOrderCollection();
 
         List<String> errors = new ArrayList<>();
-        if (CollectionUtils.isEmpty(draftDirectionOrderCollection)) {
+        if (CollectionUtils.isEmpty(judgeApprovedOrderCollection)) {
             errors.add(NO_ORDERS_IN_COLLECTION);
             return GenericAboutToStartOrSubmitCallbackResponse.<FinremCaseData>builder()
                 .data(caseData).errors(errors).build();
         }
 
+        // validate the newly uploaded orders only
+        // It does not need to verify caseDetailsBefore later as it will not show the previous uploaded orders anymore.
```

Comment:
The above comment doesn't seem to match the code below, which is verifying caseDetailsBefore to some extent.

Reply:
@ptrelease changed.